components/card/collapsible/src/index.js

```diff
@@ -31,9 +31,9 @@ class CardCollapsible extends Component {
    * @param {boolean} itemProps.highlighted Highlight the item (bold style).
    * @return {Element} Item to display on card's header info area.
    */
-  _renderInfoItem ({label, link, highlighted, target = '_blank', iconLeft: IconLeft}, index) {
+  _renderInfoItem ({label, link, highlighted, target = '_blank', iconLeft: IconLeft, onClick}, index) {
     const InfoItemElement = link ? 'a' : 'span'
-    const props = link ? { href: link, target } : {}
+    const props = link ? { href: link, target, onClick } : {}
```

Comment:
Maybe you have to add a default value: `const props = link ? { href: link, target, onClick = () => {} } : {}`

Reply:
It works properly even without a default value.